changelog/unreleased/issue-3214

```diff
@@ -0,0 +1,8 @@
+Bugfix: Treat an empty password as a fatal error for repository init.
```

Comment:
The summary line of the changelog must not end with punctuation. Please remove the trailing full stop.

Reply:
Done